data-model/src/main/java/io/micronaut/data/annotation/AutoPopulated.java

```diff
@@ -34,10 +36,24 @@
      */
     String NAME = AutoPopulated.class.getName();
 
+    /**
+     * @deprecated Replaced by {@link #UPDATABLE}
+     */
+    @Deprecated(since = "4.8", forRemoval = true)
     String UPDATEABLE = "updateable";
 
+    String UPDATABLE = "updatable";
+
     /**
      * @return Whether the property can be updated following an insert
+     * @deprecated Replaced by {@link #updatable()}
      */
+    @Deprecated(since = "4.8", forRemoval = true)
     boolean updateable() default true;
```

Comment:
maybe use `@AliasFor` so if someone sets this it sets the new one

Reply:
`would just use the old value` - I mean we alias the new one to the old one for now